Wrappers/Python/test/testclass.py

```diff
@@ -76,3 +76,35 @@ def assertDataArraysInContainerAllClose(self, container1, container2, rtol=1e-07
                     self.assertDataArraysInContainerAllClose(container1.get_item(col),container2.get_item(col), rtol=rtol,  msg=msg)
         else:
             np.testing.assert_allclose(container1.as_array(), container2.as_array(), rtol=rtol, err_msg=msg)
+
+    def assertDataContainerAllClose(self, container1, container2, rtol=1e-07, msg=None, strict=False):
+        # Test to check if two DataContainers are close, by checking
```

Comment:
add this as a docstring rather than comments

Reply:
Good point, updated